articles/liquity/Asset Risk Assessment - bLUSD.md

```diff
@@ -0,0 +1,203 @@
+
+## bLUSD accrual mechanism
+
+A user who has bonded LUSD will get nothing but a promise of redeeming accrued bLUSD at the time of chicken-in, foregoing the bonded LUSD. This accrual of bLUSD depends on the variables like time, bonded amount, chicken in fees, accrual parameter (alpha), and redemption price of bLUSD.
+
+Here is the equation:
+
+$$
+s(t) = \frac {b}{p_r}(\frac {t}{t + \alpha})
+$$
+
+$s(t)$ = Virtual accrual of bLUSD
+
+$b$ = Amount of LUSD bonded
+
+$\alpha$ = Accrual parameter
+
+$t$ = Time in seconds
+
+$p_r$ = Redemption price (LUSD in Reserve Bucket / bLUSD supply)
+
+The amount of LUSD bonded (b) is taken into consideration after reducing the 3% chicken in fees from the actual bond value for the calculation of bLUSD accrual. The accrual rate might vary based on the current redemption price (LUSD in the Reserve bucket / bLUSD supply).
+
+Moreover, there is a function where alpha is reduced to accelerate the bLUSD accrual rate. This is triggered when the weighted average bond age goes beyond the target age.
+
+![Controller logic for triggering the accrual parameter](https://lh4.googleusercontent.com/k3o7OuY1SC95kNCOCsNumC7Lx1gjbdQkyLl4HCL4-oizhUWdB9gK93Hj60Hd0ha8w_7DTDtEdPsZn3WMFSDNIDEX1_kTVOLRmtMwu2P2pOkzYM15ymjMfkJcKKtbvuQLboZMUjxHGmz-FPBTt2MmJJV-6TjarZAhM_kGG16EcnHbRhN_AIHHey4C_2DMfg)
+
+Lowering the alpha would cause the accrual of bLUSD to speed up thus bringing the break-even point and the optimal rebonding time close. This will in turn cause the average bond age to drop which is why the controller is used.
+
+## A consequence of lowering the alpha
+
+A major reason for greater bond age is people not willing to take an action (CI or CO), in other words, people are not convinced with the currently accrued bLUSD is profitable enough to chicken_in. When the [current weighted average bond age](https://dune.com/queries/1460655) is greater than the target age which is 15 days, the controller would lower the alpha.
+
+Lowering the alpha will change the non-profitable chicken_in trade to a profitable chicken_in trade by accelerating the bLUSD accrual. If we were to create a bond for 1000 LUSD on Nov 2nd the break-even time (value of accrued bLUSD = value of LUSD bonded) considering the market price of bLUSD as $1.20 turn out on Nov 25th.
+![Demonstartion of expected bond age with breeak even point and rebonding time with 1000 LUSD as on November 2nd 2022](https://lh6.googleusercontent.com/wl-7yMaLz2OLSAYuQ2vmZCFnKliQOPv5YHwXogI61kj4UM_WeXuy1rJ78K7L8c-J3fVKivEiMUvOe1mReWhDtoNG_uK5Q8tmbg-seV1gIxVvJq5CrRZr0z6mO8-vqRWUZUa7EP8eKtqgNoudnHDrUrcbKxKp7b68CPOlLNWehQ_6iNOlWbW2jl3SYziFXw)
+
+Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD. When his/her non-profitable trade is converted to a profitable one and when he/she chicken_in, the extra profit that he/she would gain will come from the bLUSD market price.
```

Comment:
use "they,their,them" as gender neutral pronoun rather than "he/she,his/her,him/her".

Reply:
> if there is only 1 user currently bonding with a considerable amount of LUSD

I am referring to a single person here so using "they", "them", "their" will not be grammatically correct as these are plural and we need a singular pronoun which I am already using.

Reply:
"They" can be used in the singular to refer to somebody of indefinite gender.
https://en.wikipedia.org/wiki/Singular_they#Contemporary_usage
It makes for less wordy writing than using he/she, him/her...

Reply:
Wow...I didn't know this. I will make the following change,

From:

> Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD. When his/her non-profitable trade is converted to a profitable one and when he/she chicken_in, the extra profit that he/she would gain will come from the bLUSD market price.

To:

> Considering a zero-sum game, if there is only 1 user currently bonding with a considerable amount of LUSD, their non-profitable trade will be converted to a profitable one (based on the mechanic discussed above). When they chicken in, the extra profit (which otherwise would not be generated if $\alpha$ was not to be lowered) will come from the bLUSD inflation.